### PR TITLE
add timeline, card event  and history page

### PR DESCRIPTION
**Pull request by Hamza-Luciano**

### CODE REVIEW

**Review by FrancoisSalmon — DISMISSED** [review]
ok for me

File: frontend/src/components/CardEvent/index.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import PropTypes from "prop-types";
+import SCardEvent from "@components/CardEvent/style";
+
+export default function CardEvent({ id, date, text }) {
+  return (
+    <SCardEvent>
+      <div className="timeline">
+        <div className={id % 2 === 0 ? "container right" : "container left"}>
+          <div className="date">{date}</div>
+          <i className="icon fa fa-home" />
```
Comment (LuigiMenez) [inline frontend/src/components/CardEvent/index.jsx:10]: ???

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Why the <i> HTML element???

**Review by FrancoisSalmon — APPROVED** [review]
ok

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
sne

**Pull request merged**